### PR TITLE
Added Indonesian Translate

### PR DESCRIPTION
**Pull request by Krindog7337**

### CODE REVIEW

File: NickvisionMoney.Shared/Resources/Strings.id.resx [hunk 1]
```diff
@@ -0,0 +1,677 @@
...
+	<data name="ResetFilters.Overview" xml:space="preserve">
+		<value>Atur Ulang Filter Gambaran</value>
+	</data>
+
+	<data name="ResetFilters.Groups" xml:space="preserve">
+		<value>Atur Ulang Filter Grup</value>
+	</data>
+
+	<data name="ResetFilters.Dates" xml:space="preserve">
+		<value>Atur Ulang FIlter Tanggal</value>
```
Comment (fsobolev) [inline NickvisionMoney.Shared/Resources/Strings.id.resx:312]: I assume there's a typo ("F**I**lter")

File: NickvisionMoney.Shared/Resources/Strings.id.resx [hunk 2]
```diff
@@ -0,0 +1,677 @@
...
+	<data name="FixErrors.WinUI" xml:space="preserve">
+		<value>Tolong perbaiki galat di atas.</value>
+	</data>
+
+	<data name="ChangelogTitle.WinUI" xml:space="preserve">
+		<value>Apa Yang Baru?</value>
+	</data>
+
+	<data name="CreditsDialogDescription.WinUI" xml:space="preserve">
+		<value>Developer:&#xA;{0}&#xA;&#xA;Designers:&#xA;{1}&#xA;&#xA;Artists:&#xA;{2}&#xA;&#xA;Translators:&#xA;{3}</value>
```
Comment (fsobolev) [inline NickvisionMoney.Shared/Resources/Strings.id.resx:630]: Not translated

File: NickvisionMoney.Shared/Resources/Strings.id.resx [hunk 3]
```diff
@@ -0,0 +1,677 @@
...
+		<value>Preferensi</value>
+	</data>
+
+	<data name="KeyboardShortcuts" xml:space="preserve">
+		<value>Pintasan Papan Tik</value>
+	</data>
+
+	<!--MainWindow-->
+	<data name="NickvisionMoneyAccount" xml:space="preserve">
+		<value>Rekening Uang Nickvision</value>
```
Comment (fsobolev) [inline NickvisionMoney.Shared/Resources/Strings.id.resx:132]: See https://github.com/nlogozzo/NickvisionMoney/pull/152#discussion_r1059423855

File: NickvisionMoney.Shared/Resources/Strings.id.resx [hunk 4]
```diff
@@ -0,0 +1,677 @@
...
+	<data name="CloseAccount.GTK" xml:space="preserve">
+		<value>Tutup Rekening</value>
+	</data>
+
+	<data name="AccountActions.GTK" xml:space="preserve">
+		<value>Tindakan</value>
+	</data>
+
+	<data name="AccountFileFilter.GTK" xml:space="preserve">
+		<value>Rekening Uang (*.nmoney)</value>
```
Comment (fsobolev) [inline NickvisionMoney.Shared/Resources/Strings.id.resx:675]: Same as "NickvisionMoneyAccount", see https://github.com/nlogozzo/NickvisionMoney/pull/152#discussion_r1059423855

**Review by fsobolev — CHANGES_REQUESTED** [review]

**Review by fsobolev — APPROVED** [review]

**Review by nlogozzo — APPROVED** [review]

**Comment by nlogozzo** [comment]
Thanks for the translation!

**Pull request merged**